test/controllers/FeedbackFormsControllerSpec.scala

```diff
@@ -332,13 +332,68 @@ class FeedbackFormsControllerSpec extends PlaySpecification with TestEnvironment
 
     "build json from case class" in new WithTestApplication {
       val questions = Question("how is knolx portal?", List("awesome", "i can do it better"))
-
       val feedbackForm = FeedbackForm("test", List(questions), active = true, BSONObjectID.parse("5943cdd60900000900409b26").get)
-
       val result = controller.jsonCountBuilder(feedbackForm)
       result must be equalTo """{"0":"2"}"""
     }
 
+    "generate feedback form preview for success" in new WithTestApplication {
+      usersRepository.getByEmail("test@example.com") returns emailObject
+
+      val feedbackForms = FeedbackForm("form name", List(Question("How good is knolx portal ?", List("1", "2", "3", "4", "5"))),
+        active = true, BSONObjectID.parse("5943cdd60900000900409b26").get)
+
+      feedbackFormsRepository.getByFeedbackFormId("5943cdd60900000900409b26") returns Future.successful(Some(feedbackForms))
+
+      val request = FakeRequest(POST, "/feedbackform/preview").withBody(Json.parse("""{"id":"5943cdd60900000900409b26"}"""))
+        .withSession("username" -> "uNtgSXeM+2V+h8ChQT/PiHq70PfDk+sGdsYAXln9GfU=")
+
+      val response = controller.getFeedbackFormPreview(request)
+      status(response) must be equalTo OK
+    }
+
+    "generate feedback form preview for failure" in new WithTestApplication {
```

Comment:
A better suggestion for this test description would be "not get feedback form because form by id does not exist". As test suite starts with "should" so the description on console would appear something like this
Feedback controller should 
"not get feedback form because form by id does not exist"